### DISCUSSION TITLE
Incorrect html display

### DISCUSSION THREAD

**Discussion by FightingFox** [structurizr/onpremises#79] [Q&A]
### Question

Good afternoon.

I use documentation creation in Lite and publishing in On-premises. Both applications are deployed in Docker. In local Lite html is displayed correctly, but On-premises displays not well. For example, in Lite \<br> is displayed as a line, but in On-premises it is literally \<br>. 

Can you help me?

**Comment by FightingFox** [comment]
@simonbrowndotje
I'm sorry for the trouble. This issue is very important to me. Can you tell me if this is my mistake or a system bug?

**Reply by simonbrowndotje** [reply]
> Can you tell me if this is my mistake or a system bug?

Impossible to say without knowing which versions you're using and some instructions to reproduce the problem you're seeing.

**Reply by FightingFox** [reply]
I use:
- Development instance on docker (build structurizr/lite:3112)
- On-premises instance in docker (build structurizr/onpremises at 6/14/23)

How reproduce the problem:
1. Create project in local Strukturizr Lite.
2. Create documentation page in this project.
3. Add HTML tags in this page (for example, ```<details>Your text</details>```).
4. Start Strukturizr Lite - I see documentation with HTML. 
5. Deploy project to On-premises - I see documentation without HTML (but I see tags).

On local:
![Снимок экрана 2023-10-23 в 15 10 16](https://github.com/structurizr/onpremises/assets/77280748/bbbcd240-c3d3-468e-9ec9-82c2401efb6c)

On On-premise:
![Снимок экрана 2023-10-23 в 15 11 08](https://github.com/structurizr/onpremises/assets/77280748/1e08111e-b67e-41c7-be05-5113e55f6dfd)

**Reply by simonbrowndotje** [reply]
I'd recommend upgrading your on-premises installation - this may resolve your issue.
